src/wp-content/themes/twentytwentyfour/functions.php

```diff
@@ -33,7 +33,7 @@ function twentytwentyfour_block_styles() {
 			array(
 				'handle' => 'twentytwentyfour-button-style-outline',
 				'src'    => get_theme_file_uri( 'assets/css/button-outline.css' ),
-				'ver'    => wp_get_theme()->get( 'Version' ),
+				'ver'    => wp_get_theme( 'twentytwentyfour' )->get( 'Version' ),
```

Comment:
Nit-pick: Maybe better to use `get_template()` instead of hard-coding `'twentytwentyfour'`?

An additional, very minor concern here is that the version always comes from TT4, while the actual `button-outline.css` file _may_ be loaded from a TT4 child theme. If we want to force that file to always be that of TT4 (which I'd argue is probably the safer call), we should replace the two `get_theme_file_*()` function calls with their equivalent `get_parent_theme_file_*()` calls. That way this functionality is consistent and will always rely on the styles from TT4, ignoring any child theme. I'd argue that makes sense here because the block styles registered below can't be overwritten by a child theme either.

I realize I was involved in the current code here, but I am now realizing the usage of `get_theme_file_*()` was probably not a good call.

Reply:
It's the end of my day, if you want to open a PR on the theme repo (or even commandeer this PR) I'll be more than happy to review early in the morning, since RC has been moved to Wednesday, but all of that sounds good to me. If you don't have the time, I'll get to it in the morning. Thank you!

Reply:
Merged and ported here!

Reply:
confirmed 👍  - https://github.com/WordPress/twentytwentyfour/pull/712 is indeed merged.